velodyne_driver/src/driver/driver.cc

```diff
@@ -92,7 +95,8 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
     }
   else
     {
-      input_.reset(new velodyne_driver::InputSocket(private_nh));
+      input_.reset(new velodyne_driver::InputSocket(private_nh,
+                                                    config_.udp_port));
```

Comment:
It looks to me like the only field in config_ that is used in InputStream is the udp_port, so I'd be hesitant to add the entire struct.